.github/pull_request_template.md

```diff
@@ -1,44 +1,5 @@
-<!-- Thank you for submitting a pull request! Find more information in our development guide at https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md and contact us on #dev in Slack. -->
-
 ## Description
 
 <!-- Elaborate beyond the title of the PR as necessary to help the reviewers and maintainers.-->
 
-<!-- Answer the following questions to help reviewers and maintainers
-understand this PR's scope at a glance:
--->
-
-> Is this change a fix, improvement, new feature, refactoring, or other?
-
-> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)
-
-> How would you describe this change to a non-technical end user or system administrator?
-
-## Related issues, pull requests, and links
-
-<!-- List any issues fixed by this PR, and provide links to other related PRs, upstream release notes, and other useful resources. For example:
-* Fixes #issuenumber
-* Related documentation in #issuenumber
-* [Some release notes](http://usefulinfo.example.com)
--->
-
-<!-- The following sections are filled in by the maintainer with input from the contributor:
-Use :white_check_mark: or (x) to signal selection.
--->
-
-## Documentation
-
-( ) No documentation is needed.
-( ) Sufficient documentation is included in this PR.
-( ) Documentation PR is available with #prnumber.
-( ) Documentation issue #issuenumber is filed, and can be handled later.
-
-## Release notes
```

Comment:
I think release notes section is important reminder to add that to PR desc. 

I do not care much about the rest.

Reply:
Documentation + Release notes headers are useful and should be kept. Everything else often isn't very useful.

Reply:
>  Everything else often isn't very useful.

Good that we have agreement about this.

Re release notes -- 
i think that, at the moment of filing PR, it's hard to formulate good release notes. The PR content may change over time.

anyway, I think it's might be useful if it is filled in. -- Is it?

Reply:
> Documentation + Release notes headers are useful and should be kept. Everything else often isn't very useful.

not sure what you'd want to _remove_?

Reply:
> The boilerplate under "Description" and entirety of "Related issues, pull requests, and links".

I wish we could remove much more, but let's do baby steps.
I restored all other parts for now.
@hashhar PTAL